### ISSUE TITLE
[med] Color Contrast: Many Pages: Grey icons against white background fail contrast requirements

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5506]
## Action Performed:
Prerequisite: user is logged in with valid credentials.
1. Using Chrome, open https://staging.new.expensify.com/
2. Click on user icon from the top left.
3. Measure the color contrast ratio of the grey icons against white background.

## Expected Result:
The visual presentation of meaningful graphical objects or interactive components should have a contrast ratio of at least 3.00:1 with the surrounding background color.

## Actual Result:
The grey icons (#C6C9CA) on white background (#FFFFFF) have a color contrast ratio of 1.66:1.

Other occurrences:
1. Request Money -> The grey close button (#C6C9CA) against white background (#FFFFFF) has a color contrast ratio of 1.66:1.
2. Split bill -> The grey close button (#C6C9CA) against white background (#FFFFFF) has a color contrast ratio of 1.66:1.
3. New chat overlay -> The grey close button (#C6C9CA) against white background (#FFFFFF) has a color contrast ratio of 1.66:1.
4. Change password -> The grey close button (#C6C9CA) against white background (#FFFFFF) has a color contrast ratio of 1.66:1.
5. Preferences overlay -> The grey back and close buttons (#C6C9CA) against white background (#FFFFFF) have a color contrast ratio of 1.66:1.
6. New group overlay -> The grey close button (#C6C9CA) against white background (#FFFFFF) has a color contrast ratio of 1.66:1.
7. Chat -> All action button icons for messages, all icons in the message bar, call and pin buttons at top of chat, search icon next to profile button are grey (#C6C9CA) resulting in a color contrast ratio of 1.66:1 against white background.

## Workaround:
No workaround

## Area issue was found in:
Settings overlay, Request money overlay, Split bill overlay, New chat overlay, Change password overlay, Preferences overlay, New group overlay, Chat page

## Failed WCAG checkpoints
1.4.11

## User impact:
The low-vision and color-blind users might not be able to see the icons.

## Suggested resolution:
Use a darker shade of grey for the icons to achieve a color contrast ratio of at least 3.00:1. One example of color to use is `#8D9596`, which will give a color contrast ratio of 3.05:1.

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5182543_Change_password-Grey_close_button_fails_color_contrast_against_white_background](https://user-images.githubusercontent.com/88733897/134723017-d9494789-3c75-4e27-bf2c-04be7cd004e0.png)
![Bug5182543_Chat_-_grey_icons_on_white_background_fail_contrast_requirements](https://user-images.githubusercontent.com/88733897/134723025-69845cf0-5cce-4455-a1f7-6fcac4344d51.png)
![Bug5182543_Color_Contrast-Create_New_Group-Grey_close_icon_on_white_background_fails_contrast](https://user-images.githubusercontent.com/88733897/134723033-dbab57f9-f5ab-46a3-acf4-e0da4e90d540.png)
![Bug5182543_Grey_icons_fail_color_contrast](https://user-images.githubusercontent.com/88733897/134723040-aa4d136e-aac1-4d24-b84c-f8056f800264.png)
![Bug5182543_New_chat-Grey_close_button_fails_color_contrast_against_white_background](https://user-images.githubusercontent.com/88733897/134723051-efd8264b-4955-4d2b-8f4b-c4f62463212b.png)
![Bug5182543_Preferences_-_grey_icons_on_white_background_fail_contrast_requirements](https://user-images.githubusercontent.com/88733897/134723058-36519618-3916-4455-a650-035979f28e06.png)
![Bug5182543_Request_money-Grey_close_button_fails_color_contrast_against_white_background](https://user-images.githubusercontent.com/88733897/134723065-ee322587-e9b8-44f8-a31f-b631fe18a380.png)
![Bug5182543_Split_bill-Grey_close_button_fails_color_contrast_against_white_background](https://user-images.githubusercontent.com/88733897/134723073-80037469-2a40-498e-837a-04c4734055c3.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
